WireDomain/Sources/WireDomain/Event Decryption/ProteusMessageDecryptorError.swift

```diff
@@ -0,0 +1,28 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
```

Comment:
@caldrian would say the import is not needed 😅

Reply:
Is it always implicit? It's always there when I create a file so I just leave it. 🤷🏻‍♂️

Reply:
removed 31a6fbe7096f2af8ceb7eef257ed1ff75fccaea9

Reply:
Thanks and sorry for the late reply. Foundation is just needed for types (like UUID) that are not included in Swift itself (String, etc.).